backend/open_webui/utils/middleware.py

```
@@ -292,6 +296,45 @@ async def tool_call_handler(tool_call):
+                    "k": 3,
+                }
+            ),
+            user,
+        )
+    except Exception as e:
+        log.debug(e)
+        results = None
+
+    user_context = ""
+    if results and hasattr(results, "documents"):
+        if results.documents and len(results.documents) > 0:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)